.github/workflows/feed.yml

```
@@ -82,3 +82,39 @@ jobs:
         type: 'stream'
         topic: 'New package registrations'
         content: ${{ steps.zulip.outputs.content }}
+        
+  update-twiter:
+      if: ${{ github.event.label.name == 'new package' && github.repository == github.event.pull_request.head.repo.full_name }}
+      runs-on: ubuntu-latest
+      steps:
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: 1.6.2
+      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
```

Latest stable version is 3.0.2:
```suggestion
      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v3.0.2
```